src/Main.java

```diff
@@ -1,5 +1,76 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        //Задача 1
+        int age = 18;
+        if (age >= 18) {
+            System.out.println("Если возраст человека равен " + age + ", то он совершеннолетний");
+        }
+        if (age < 18) {
+            System.out.println("Если возраст человека равен " + age + ", то он не достиг совершеннолетия");
+        }
+
+        //Задача 2
+        int t = 6;
+        if (t > 5) {
+            System.out.println("Сегодня тепло, можно идти без шапки");
+        }
+        if (t < 5) {
+            System.out.println("На улице холодно, нужно надеть шапку");
+        }
+
+        //Задача 3
+        int speed = 90;
+        if (speed > 60) {
+            System.out.println("Если скорость " + speed + ", то придется заплатить штраф");
+        } else {
+            System.out.println("Если скорость " + speed + ", то можно ездить спокойно");
+        }
+
+        //Задача 4
+        int age1 = 4;
+        if (age1 >= 2 && age1 <= 6) {
+            System.out.println("Если человеку " + age1 + ", то ему нужно ходить в детский сад");
+        } else if (age1 <= 17) {
+            System.out.println("Если человеку " + age1 + ", то ему нужно ходить в школу");
+        } else if (age1 <= 24) {
+            System.out.println("Если человеку " + age1 + ", то ему нужно ходить в университет");
+        } else {
+            System.out.println("Если человеку " + age1 + ", то ему нужно ходить на работу");
+        }
+
+        //Задача 5
+        int age2 = 5;
```

Comment:
можно было не создавать новую переменную с "плохим" названием (цифра на конце), а просто переиспользовать уже созданную переменную
```suggestion
        age = 5;
```